src/cli/manager.rs

```diff
@@ -83,6 +88,14 @@ fn get_clap_matches<'a>() -> clap::ArgMatches<'a> {
                 .takes_value(true)
                 .default_value("udpout:0.0.0.0:14550"),
         )
+        .arg(
+            clap::Arg::with_name("www-path")
+                .long("www-path")
+                .value_name("NAME")
+                .help("Sets the WWW path")
+                .takes_value(true)
+                .default_value("/opt/blueos/mavlink-camera-manager/www"),
```

Comment:
That's a good approach, but since we are proving a zip file with the www folder there, we should use by default the current binary path, with that anyone that downloads the zip and execute the binary will get it working without moving files around.